### PR TITLE
Timid women/men costumes!

### PR DESCRIPTION
**Pull request by BlueWildrose**
## About The Pull Request

image
This will add timid women/men costumes into the game.
These are simply cute costumes, nothing more. Clearly.
A total of six new items (eight if you count the hood objects) have been added, and a new pair of socks that are selectable from the character creation menu or from a dresser.
The new objects are the following:
**Timid woman/man hoodie:** Polychromic suit, belt loop and buckle, toggleable hoodie. I pretty much just copypasted the code from the polychromic winter coat.
**Timid woman/man mask:** Functions identically to the mime/clown mask.
**Timid woman/man shoes:** Polychromic shoes, only one tone.
The socks: Garters. They're intended to complement the timid woman outfit in full if someone wants to fully dress up like the intended outfit, but otherwise it's available for everyone to use.
image

Recommended to use a gear harness with the timid outfit.

These costume pieces have been added to the **Autodrobe** vendor.
Additionally, if you're a **mime**, you can **include the costume pieces into your loadout** for a total of 3 points, so you can start with the costume. Make it your part of your gimmick as a mime that way!

_Known issues:_ The inventory icon when the hoodie is toggled up will look ugly. This is also an issue with the polychromic winter coat as well.

## Why It's Good For The Game

More costumes means more fun. More clothing is always fun.

## Changelog
:cl:
add: Adds timid women/men costumes
add: Adds said above costumes to the autodrobe vendor
add: The costume pieces are added to the loadout - exclusive to the Mime job.
add: New pair of socks: Garters
imageadd: Obviously, adding a costume means adding in new sprites; a total of 12 DMI files have been modified.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
why is the male belt and mask neon pink and green

**Comment by BlueWildrose** [comment]
> 
> 
> why is the male belt and mask neon pink and green

Just me demonstrating polychromic stuff. Obviously better options can be picked. It defaults to brown and yellow, just like what's presented on the floor and on the female one to the left. I just picked those colors for the preview because it just felt like it stood out more for demonstration purposes - because bad color choices will definitely be noticeable.

**Comment by BlueWildrose** [comment]
oh great, conflicts - was some other clothing added recently that I missed?

File: code/modules/mob/dead/new_player/sprite_accessories/socks.dm [hunk 1]
```diff
@@ -10,6 +10,9 @@
 	icon_state = null
 
 // please make sure they're sorted alphabetically and categorized
+/datum/sprite_accessory/underwear/socks/garter
+	name = "Garter"
+	icon_state = "garter"
```
Comment (Ghommie) [inline code/modules/mob/dead/new_player/sprite_accessories/socks.dm:15]: Being grayscale it could also have `has_color = TRUE`

**Review by Ghommie — CHANGES_REQUESTED** [review]
Merge conflicts.

**Comment by Iamgoofball** [comment]
i dont understand, are these like tentacle monster shy guys?

**Comment by Ghommie** [comment]
It's their snowflake character's tails.

**Comment by Iamgoofball** [comment]
Understandable carry on

**Pull request closed without merging**